libraries/freertos_plus/standard/freertos_plus_tcp/source/portable/NetworkInterface/STM32Fxx/NetworkInterface.c

```diff
@@ -940,6 +999,8 @@ uint8_t *pucBuffer;
 		pxDMARxDescriptor->ControlBufferSize = ETH_DMARXDESC_RCH | (uint32_t)ETH_RX_BUF_SIZE;  
 		pxDMARxDescriptor->Status = ETH_DMARXDESC_OWN;
 
+		/* Ensure completion of memory access */
+		__DSB();
```

Comment:
See previous comment about DMA descriptor handling macro

Reply:
I will come back later to this.